tools/createSovereignGenesis/create-sovereign-genesis.ts

```diff
@@ -145,6 +188,56 @@ async function main() {
         outWETHAddress = wethObject.address;
     }
 
+    // check preMintAddress is an address
+    if (createGenesisSovereignParams.setPreMintAccount === true) {
+        logger.info('Add preMintAccount');
+        if (ethers.isAddress(createGenesisSovereignParams.preMintAccount.address) == false) {
```

Comment:
Maybe add this check at the beginning with other params checks

Reply:
yep!